package.json

```diff
@@ -50,15 +53,18 @@
     "build": "webpack --mode production ",
     "cy:open": "cypress open",
     "cy:run": "cypress run --headless",
-    "cy:test": "npm run build && start-server-and-test start http://localhost:8080 cy:run",
+    "cy:test": "yarn build && start-server-and-test start http://localhost:8080 cy:run",
     "doc": "jsdoc -p -r -c jsdoc_conf.json src -d doc README.md",
     "format": "prettier --write 'src/**/*.js' && eslint 'src/**/*.js' --fix && stylelint 'style/**/*.css' 'src/**/*.css' 'src/**/*.scss' --fix",
     "lint": "eslint 'src/**/*.js' && stylelint 'style/**/*.css' 'src/**/*.css' 'src/**/*.scss'",
-    "publish:beta": "npm run publish:public --tag beta",
-    "publish:public": "npm version && npm run build && git push origin HEAD && npm publish",
-    "start": "npm run build && webpack-dev-server --mode development --open",
+    "publish:beta": "yarn release -- --release-as patch --prerelease beta && yarn build && git push origin HEAD && yarn publish --tag beta",
+    "publish:major": "yarn release && yarn build && git push origin HEAD && yarn publish",
```

Comment:
Do we really need the `--release-as` flag and NPM scripts? As far as I understand this should be handled by `standard-version` and only used in special situations. This also makes beta releases limited to patch releases.

Reply:
You're right, I didn't realise it detects the version hierarchy using the commit type. Have updated this in the meantime